feat: block_structure.storage_backing_for_cache toggle removed depr32

Yagnesh1998 opened the pull request:
**Description**
As Per this DEPR: https://github.com/openedx/public-engineering/issues/32

Removing: Toggle labeled 'block_structure.storage_backing_for_cache'. The toggle has the following annotation as a description:

When enabled, block structures are stored in a more permanent storage, like a database, which provides an additional backup for cache misses, instead having them regenerated. The regeneration of block structures is a time consuming process. Therefore, enabling this switch is recommended for Production.

**Supporting information**
as per the original jeera issue https://openedx.atlassian.net/browse/DEPR-145, this toggle is removed

jajjibhai008 committed:
feat: add hook to modify courserun data for executive education courses

Commit:
feat: block_structure.storage_backing_for_cache toggle removed depr32

openedx-webhooks commented:
Thanks for the pull request, @Yagnesh1998!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.




<!-- comment:external_pr -->
<!-- data: eyJkcmFmdCI6IGZhbHNlfQ== -->

Commit:
feat: update test_dump_to_neo4j.py

Commit:
feat: update test_api.py

Commit:
feat: update store.py

Commit:
feat: update common.py

Commit:
feat: update test_api.py

awais786 committed:
feat!: upgrading django-storages to 1.10.1 (#32571)

* feat!: upgrading django-storages to 1.10.1

UsamaSadiq committed:
fix: fix django 41 deprecation warnings (#33099)

awais786 committed:
Revert "feat!: upgrading django-storages to 1.10.1 (#32571)" (#33109)

This reverts commit 9281ea4a9a11719db8feee1d2b256fff556a6ef3.

awais786 committed:
Revert "Revert "feat!: upgrading django-storages to 1.10.1 (#32571)" (#33109)" (#33111)

This reverts commit 191b825a2ce982cd8c1f3e560f1f69f4cf2e3535.

kiram15 committed:
feat: adding unenrollments to event bus (#33085)

* feat: adding unenrollments to event bus

* fix: quality fixes

* fix: tweaks to pass tests

* fix: more tweaks for testing

---------

Co-authored-by: John Nagro <jnagro@edx.org>

bradenmacdonald committed:
fix: incorrect type hints in a few places (#33104)

Commit:
chore: version bump

Yagnesh1998 committed:
feat: Update common.py

awais786 committed:
feat!: upgrading `django-storages` to `1.11.1` (#33115)

* feat: upgrading `django-storages` to `1.11.1`

Commit:
feat: update test_api.py

irtazaakram committed:
fix: func.__name__ issue (#33133)

irtazaakram committed:
fix: object has no attribute is_ajax (#33134)

edx-requirements-bot committed:
chore: Updating Python Requirements (#33128)

awais786 committed:
Revert "Revert "Revert "feat!: upgrading django-storages to 1.10.1 (#32571)" (#33109)" (#33111)" (#33113)

This reverts commit 2f8475c307edd6a60711cbd0be22dda108f55a3e.

johnnagro committed:
feat: release edx-enterprise 4.1.4 (#33135)

KristinAoki committed:
feat: add API endpoint for asset usage search (#33092)

alex-sheehan-edx committed:
feat: Upgrade Python dependency edx-enterprise

enterprise version bump

Commit generated by workflow `openedx/edx-platform/.github/workflows/upgrade-one-python-dependency.yml@refs/heads/master`

Commit:
chore: geoip2: update maxmind geolite country database

awais786 committed:
feat!: upgrading django-storages to 1.11.1

muhammadadeeltajamul committed:
feat: added new_question_post and new_discussion_post notification (#33043)

Commit:
feat: update generate_course_blocks.py

Commit:
feat: update test_generate_course_blocks.py

muhammadadeeltajamul committed:
feat: make notification type info translateable (#33071)

muhammadadeeltajamul committed:
fix: handle new_comment notification edge case (#33088)

awais786 committed:
test: adding check to run migrations tests only if its django32. (#33141)

johnnagro committed:
fix: better username lookup in tpa pipeline (#33145)

kdmccormick committed:
build: don't exclude lms/static/css/vendor from Docker build (#33143)

The .dockerignore file smartly excludes lms/static/css from the Docker
build, as most of its contents are generated CSS.

However, there is one version-controlled subdirectory of lms/static/css:
lms/static/css/vendor.

In order to correctly build a Docker image using a bind-mounted
edx-platform repository, we need this vendor directory to be included.

kiram15 committed:
fix: bump edx-enterprise (#33149)

Commit:
feat: update lms_common.py

Commit:
feat update lms_common.py

Yagnesh1998 committed:
Merge branch 'master' into edx-depr32

openedx-webhooks commented:
@Yagnesh1998 Even though your pull request wasn’t merged, please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/edx-platform&entry.867055334=https://github.com/openedx/edx-platform/pull/33108&entry.1484655318=https://github.com/Yagnesh1998&entry.752974735=2023-08-28+06:48&entry.1917517419=2023-09-01+10:12&entry.2133058324=No) so we can improve your experience in the future.

<!-- comment:end_survey -->
<!-- data: eyJqaXJhLXByZS1jbG9zZSI6IG51bGx9 -->
